### PR TITLE
Update missing Chinese translation about cis、istio、monitoring and logging parts for latest and v2.8 

### PR DESCRIPTION
**Pull request by GGGitBoy**
Update part of Chinese missing docs for the latest and version 2.8 which includes:

- cis-scan
- istio
- monitoring
- logging

### CODE REVIEW

**Comment by GGGitBoy** [comment]
@JacieChao Please help me review this PR. Thank you.

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/logging/logging.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+```bash
+# Cluster Management UI
+https://<your-url>/c/<cluster-id>/
+
+# Cluster Dashboard
+https://<your-url>/dashboard/c/<cluster-id>/
+```
+
+现在我们有了 `<cluster-id>` 命名空间，我们可以删除导致 `cattle-logging` 不断重新创建的自定义资源。
+*警告*：请当前未使用确保 Logging（从旧版 Rancher UI 全局视图中安装的版本）。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/logging/logging.md:110]: 请确保当前未使用 Logging （从旧版 Rancher UI 全局视图中安装的版本）

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/integrations-in-rancher/monitoring-and-alerting"/>
+</head>
+
+你可以使用 `rancher-monitoring` 应用，将业界领先的开源监控和告警解决方案快速部署到你的集群中。
+
+在 Rancher v2.5 中引入的 `rancher-monitoring` operator 由 [Prometheus](https://prometheus.io/)、[Grafana](https://grafana.com/grafana/)、[Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/)， [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) 和 [Prometheus adapter](https://github.com/DirectXMan12/k8s-prometheus-adapter) 提供支持。
+
+有关在 Rancher v2.2 到 v2.4 中可用的 V1 监控和警报的信息，请参阅 Rancher v2.0 到 v2.4 文档中的[集群监控](/versioned_docs/version-2.0-2.4/explanations/integrations-in-rancher/cluster-monitoring/cluster-monitoring.md)，[告警](/versioned_docs/version-2.0-2.4/explanations/integrations-in-rancher/cluster-alerts/cluster-alerts.md)，[通知](/versioned_docs/version-2.0-2.4/explanations/integrations-in-rancher/notifiers.md)和[工具](/versioned_docs/version-2.0-2.4/reference-guides/rancher-project-tools/rancher-project-tools.md)。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md:14]: V1 监控和警报 => V1 监控和告警

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+使用 `rancher-monitoring` 应用程序，你可以快速部署领先的开源监控和警报解决方案到你的集群上。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md:16]: 开源监控和警报 => 开源监控和告警

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+有关监控组件如何协同工作的说明，请参阅 [Monitoring 工作原理](how-monitoring-works.md)。
+
+## 默认组件和部署
+
+### 内置仪表板
+
+默认情况下，监控应用将 Grafana 仪表板（由 [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) 项目策划）部署到集群上。
+
+它害部署一个 Alertmanager UI 和一个 Prometheus UI。有关这些工具的更多信息，请参见[内置仪表板](built-in-dashboards.md)。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md:41]: 害 => 还

File: i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+有关 Rancher UI 中公开的监控组件列表，以及编辑它们的常见用例，请参阅[本节](how-monitoring-works.md#rancher-ui-中公开的组件)。
+
+## RBAC
+
+有关配置 monitoring 访问权限的信息，请参阅[此页面](rbac-for-monitoring.md)。
+
+:::note
+
+Rancher 和 Project 的读取权限并不一定适用于监控相关资源. 查看 [monitoring-ui-view](rbac-for-monitoring.md#additional-monitoring-clusterroles) 获取更多详细信息.
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/integrations-in-rancher/monitoring-and-alerting/monitoring-and-alerting.md:63]: 这里的url应该是 `rbac-for-monitoring.md#其他监控角色`

**Review by JacieChao — COMMENTED** [review]
Please fix the problem for both latest and v2.8

**Review by JacieChao — COMMENTED** [review]
LGTM

**Comment by JacieChao** [comment]
To avoid the 404 problem which is caused by some missing CN documents. The PR will be ready after other relative docs are ready:
- rancher-security
- installation-and-upgrade
- kubernetes-clusters-in-rancher-setup

**Pull request merged**